resources/testdriver.js

```diff
@@ -1100,6 +1100,14 @@
             throw new Error("get_named_cookie() is not implemented by testdriver-vendor.js");
         },
 
+        async get_computed_role(element) {
+            throw new Error("get_computed_role is a testdriver.js function which cannot be run in this context.");
```

Comment:
I think the message should be consistent with the others.

Reply:
In this case, I think it's not related to whether or not testdriver-vendor.js has implemented the function, so I wasn't sure what error to put. Maybe some even more generic:

```
get_computed_role() is not implemented in this context.
```